src/ui/webeiddialog.cpp

```diff
@@ -99,6 +101,10 @@ WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Private)
                                        ui->pinEntryTimeoutProgressBar->maximum());
     connect(ui->pinTimeoutTimer, &QTimeLine::frameChanged, ui->pinEntryTimeoutProgressBar,
             &QProgressBar::setValue);
+    connect(ui->pinTimeoutTimer, &QTimeLine::frameChanged, ui->pinTimeRemaining, [this](int value) {
+        ui->pinTimeRemaining->setText(
+            tr("Time remaining: <b>%1</b>").arg(ui->pinEntryTimeoutProgressBar->maximum() - value));
```

Comment:
Should we convert this value to minutes & seconds?

Reply:
It is seconds